### ISSUE TITLE
Error: useBlockTime:: Number can only safely store up to 53 bits

### ISSUE THREAD

**Issue by discovol** [polkadot-js/apps#7163]
Uncaught error. Something went wrong with the query and rendering of this component. Please supply all the details below when logging an issue, it may help in tracing the cause.

useBlockTime:: Number can only safely store up to 53 bits:: Error: Number can only safely store up to 53 bits at n (https://polkadot.js.org/apps/polkadot.02.ae863b30.js:1:110437) at i.toNumber (https://polkadot.js.org/apps/polkadot.02.ae863b30.js:1:116715) at https://polkadot.js.org/apps/main.43da798b.js:2:468888 at Object.useMemo (https://polkadot.js.org/apps/react.01.0a633bcc.js:2:319752) at t.useMemo (https://polkadot.js.org/apps/react.01.0a633bcc.js:2:606685) at https://polkadot.js.org/apps/main.43da798b.js:2:468470 at https://polkadot.js.org/apps/main.43da798b.js:2:338625 at https://polkadot.js.org/apps/main.43da798b.js:2:469412 at aa (https://polkadot.js.org/apps/react.01.0a633bcc.js:2:313829) at $s (https://polkadot.js.org/apps/react.01.0a633bcc.js:2:365878)

Error: useBlockTime:: Number can only safely store up to 53 bits:: Error: Number can only safely store up to 53 bits
at n (https://polkadot.js.org/apps/polkadot.02.ae863b30.js:1:110437)
at i.toNumber (https://polkadot.js.org/apps/polkadot.02.ae863b30.js:1:116715)
at https://polkadot.js.org/apps/main.43da798b.js:2:468888
at Object.useMemo (https://polkadot.js.org/apps/react.01.0a633bcc.js:2:319752)
at t.useMemo (https://polkadot.js.org/apps/react.01.0a633bcc.js:2:606685)
at https://polkadot.js.org/apps/main.43da798b.js:2:468470
at https://polkadot.js.org/apps/main.43da798b.js:2:338625
at https://polkadot.js.org/apps/main.43da798b.js:2:469412
at aa (https://polkadot.js.org/apps/react.01.0a633bcc.js:2:313829)
at $s (https://polkadot.js.org/apps/react.01.0a633bcc.js:2:365878)
at https://polkadot.js.org/apps/main.43da798b.js:2:338648
at https://polkadot.js.org/apps/main.43da798b.js:2:469412
at aa (https://polkadot.js.org/apps/react.01.0a633bcc.js:2:313829)
at $s (https://polkadot.js.org/apps/react.01.0a633bcc.js:2:365878)
at Bl (https://polkadot.js.org/apps/react.01.0a633bcc.js:2:353152)
at Pl (https://polkadot.js.org/apps/react.01.0a633bcc.js:2:353080)
at Sl (https://polkadot.js.org/apps/react.01.0a633bcc.js:2:352943)
at ml (https://polkadot.js.org/apps/react.01.0a633bcc.js:2:349930)
at https://polkadot.js.org/apps/react.01.0a633bcc.js:2:299751
at t.unstable_runWithPriority (https://polkadot.js.org/apps/other.01.8f9552e1.js:2:86642)

**Comment by jacogr** [comment]
Which chain?

**Comment by discovol** [comment]
https://polkadot.js.org/apps/ & wss://discovol.network 

https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fdiscovol.network#/accounts

**Comment by jacogr** [comment]
Ok, the chain has no types added to the apps UI, so this is expected. when it it listed the team should supply their type overrides. (It is quite possibly something amis in the types for either the blockNumber or possibly a different configuration)

As it stands I get no issues when connecting though to that link. Accounts/explorer/etc. are all fine (so it could also be local types injected on your side that are incorrect)

TL;DR Cannot replicate, when listed the chain should supply the correct types for their setup

**Comment by jacogr** [comment]
It could be due to an excessively large lock (`useBlockTime` takes blocks to days), but the only way to verify is with an affected address (which then can be checked in the address tab to replicate)
